tests/kafkatest/tests/streams/streams_broker_bounce_test.py

```diff
@@ -152,16 +151,9 @@ def confirm_topics_on_all_brokers(self, expected_topic_set):
         
     def setup_system(self, start_processor=True, num_threads=3):
         # Setup phase
-        self.zk = (
-            ZookeeperService(self.test_context, 1)
-            if quorum.for_test(self.test_context) == quorum.zk
-            else None
-        )
-        if self.zk:
-            self.zk.start()
-
-        self.kafka = KafkaService(self.test_context, num_nodes=self.replication, zk=self.zk, topics=self.topics,
-                                  controller_num_nodes_override=1)
+
+        self.kafka = KafkaService(self.test_context, num_nodes=3, zk=None, topics=self.topics,
```

Comment:
Would it make sense to change `KafkaService` to use `zk=None` as default so we don't have to pass it at all any longer? (Of course, touching `KafkaService` might affect _all_ system tests, so not sure what the impact / splash radius would be? -- On the other side, if `zk` is mandatory, making it optional with default `None` might not affect any existing test after all?)

Reply:
Why changing `num_nodes=self.replication` to `num_nodes=3` ?

Reply:
> Would it make sense to change KafkaService to use zk=None as default so we don't have to pass it at all any longer? (Of course, touching KafkaService might affect all system tests, so not sure what the impact / splash radius would be? -- On the other side, if zk is mandatory, making it optional with default None might not affect any existing test after all?)

I'll leave it as is.  the `zk` parameter has no default value in the `KafkaService` `__init__` method.  Parameters without default values must come before the ones that do have them, so to make this change will update the parameter order which I believe will affect all other system tests.

Reply:
> Why changing num_nodes=self.replication to num_nodes=3 ?

It never changes, so I thought I'd set explicitly, but there's no harm in putting it back.